app/views/course/assessment/answer/_worksheet.html.slim

```diff
@@ -1,16 +1,22 @@
 - if answer.submission.attempting? && answer.specific.is_a?(Course::Assessment::Answer::Programming)
+  - programming_answer = answer.specific
+  - if answer.question.actable.attempt_limit
+    - button_text = t('.run_code_with_limit', limit: programming_answer.attempting_times_left)
+  - else
+    - button_text = t('.run_code')
```

Comment:
Sorry 1 more thing here - should we render `t('.run_code')` for staff as well? Don't think it makes sense to display how many attempts they have left, and they are able to still run with 0 attempts left. 

If so, we can simplify `enable_submit_button?` as well.